vowel-count-task.js

```diff
@@ -0,0 +1,10 @@
+// Napisz funkcję vowelCount(), która zwróci liczbę samogłosek w podanym łańcuchu
+// znaków.
+
+function vowelCount(string) {
+    const array = string.match(/[aeiouy]/gi);
```

Comment:
`array` is a terrible name for a variable. It says nothing about what's inside. Check in the documentation what the `match` method returns and name the variable reasonable. For array we use a plural form of adverts. No need for the `array` part. `cars`, `posts` etc. it enough.